src/js/messaging/containers/Folder.jsx

```diff
@@ -236,9 +243,11 @@ export class Folder extends React.Component {
             isAdvancedVisible={this.props.isAdvancedVisible}
             onAdvancedSearch={this.props.toggleAdvancedSearch}
             onDateChange={this.props.setDateRange}
-            params={this.props.searchParams}
+            onError={this.props.openAlert}
+            onErrorClose={this.props.closeAlert}
```

Comment:
Sure. I'll remove it. Anything else I should change before I re-assign it for review?